src/runtime/callbacks.js

```diff
@@ -144,50 +144,45 @@ export class Callbacks {
    * @param {function(!Promise<!../api/subscribe-response.SubscribeResponse>)} callback
    */
   setOnSubscribeResponse(callback) {
-    this.setCallback_(CallbackId.SUBSCRIBE_RESPONSE, callback);
+    console./*OK*/ warn(
```

Comment:
add a warn method to log.js and call that rather than putting the console.warn here (ditto below)

Reply:
Done